components/commercial_filter/filters/paginator/paginator_view.coffee

```diff
@@ -22,10 +22,17 @@ module.exports = class PaginatorView extends Backbone.View
 
   totalPages: ->
     calculated = Math.ceil(@filter.get('total') / @params.get('size'))
-    total = Math.min calculated, @maxPage
+    Math.min calculated, @maxPage
+
+  adjustedTotal: ->
+    Math.max @totalPages(), 2
 
   render: ->
     @$el.html template
       current: parseInt @params.get('page')
-      total: @totalPages()
+      total: @adjustedTotal()
```

Comment:
We always want to have a minimum of 2 pages, but we also want to know how many total there _actually_ are, so we can manually hide those.